lib/search/ui/search_screen.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttercon/common/data/models/local/local_session.dart';
+import 'package:fluttercon/common/data/models/local/local_speaker.dart';
+import 'package:fluttercon/common/widgets/resolved_image.dart';
+import 'package:fluttercon/core/di/injectable.dart';
+import 'package:fluttercon/common/utils/router.dart';
+import 'package:fluttercon/search/cubit/search_cubit.dart';
+import 'package:go_router/go_router.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+import '../cubit/search_state.dart';
+import '../models/search_result.dart';
+
+class SearchScreen extends StatelessWidget {
+  const SearchScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => getIt<SearchCubit>(),
```

Comment:
Won't be needed this way once you register this in `lib/bootstrap.dart`

Reply:
We won't this file actually, [lib/search/ui/search_screen.dart](https://github.com/droidconKE/flutterconKEApp/pull/132/files/db64ebaccaa91908e99241b5ed96613909204903#diff-2e38e0f8cfe387c52232413a31c2630fe746363c14da09bf44b9c6e7352ca171),
since the results now are displayed on the homescreen using  [lib/search/ui/widgets/search_bar.dart](https://github.com/droidconKE/flutterconKEApp/pull/132/files/db64ebaccaa91908e99241b5ed96613909204903#diff-9c8ee8ac7a6a8c64bc86196f49eee3c455382cf7ed89a85488dba33c7ccd7a9b)